Hakawai/Mentions/HKWMentionsPlugin.h

```diff
@@ -17,11 +17,34 @@
 #import "HKWChooserViewProtocol.h"
 #import "HKWMentionsDefaultChooserViewDelegate.h"
 
+static NSString* _Nonnull const HKWMentionAttributeName = @"HKWMentionAttributeName";
+
+// Don't confuse this with the public 'HKWMentionsPluginState', which exposes fewer implementation details.
+typedef NS_ENUM(NSInteger, HKWMentionsState) {
```

Comment:
so in simple refactor. this is the only state we will be managing?

Reply:
removed this. we don't need it